bee.php

```diff
@@ -41,9 +41,19 @@
  *
  * @see https://www.php.net/manual/en/function.set-error-handler.php
  */
-function bee_error_handler($errno, $message, $filename, $line, $context) {
+function bee_error_handler($errno, $message, $filename, $line, $context = []) {
```

Comment:
Backdrop (and Bee) still use the long array syntax, so this should be `array()` instead.